tests/ui/treat-err-as-bug/span_delayed_bug.stderr

```diff
@@ -4,6 +4,8 @@ error: internal compiler error: delayed bug triggered by #[rustc_error(delayed_b
 LL | fn main() {}
    | ^^^^^^^^^
 
+
+
```

Comment:
Are those for empty messages? in that case we can maybe avoid the extra newlines.

Reply:
I don't see how this changed nor why it doesn't currently print the full panic message.

Reply:
Ah, the tests use output normalization which strips the thread panic message.